### ISSUE TITLE
What could be causing loss to be nan?

### ISSUE THREAD

**Issue by veioxve3r1d23seb2g6b** [zhouhaoyi/Informer2020#193]
Use GPU: cuda:0
>>>>>>>start training : informer_custom_ftMS_sl96_ll48_pl24_dm512_nh8_el2_dl1_df2048_atprob_fc5_ebtimeF_dtTrue_mxTrue_exp_0>>>>>>>>>>>>>>>>>>>>>>>>>>
train 1199
val 166
test 353
Epoch: 1 cost time: 3.8804690837860107
Epoch: 1, Steps: 9 | Train Loss: nan Vali Loss: nan Test Loss: nan
Validation loss decreased (inf --> nan).  Saving model ...
Updating learning rate to 1e-06
Epoch: 2 cost time: 3.8711836338043213
Epoch: 2, Steps: 9 | Train Loss: nan Vali Loss: nan Test Loss: nan
Validation loss decreased (nan --> nan).  Saving model ...
Updating learning rate to 5e-07
Epoch: 3 cost time: 3.870296001434326
Epoch: 3, Steps: 9 | Train Loss: nan Vali Loss: nan Test Loss: nan
Validation loss decreased (nan --> nan).  Saving model ...
Updating learning rate to 2.5e-07
Epoch: 4 cost time: 3.8688690662384033
Epoch: 4, Steps: 9 | Train Loss: nan Vali Loss: nan Test Loss: nan
Validation loss decreased (nan --> nan).  Saving model ...
Updating learning rate to 1.25e-07
Epoch: 5 cost time: 3.8895139694213867
Epoch: 5, Steps: 9 | Train Loss: nan Vali Loss: nan Test Loss: nan
Validation loss decreased (nan --> nan).  Saving model ...
Updating learning rate to 6.25e-08
Epoch: 6 cost time: 3.902830123901367
Epoch: 6, Steps: 9 | Train Loss: nan Vali Loss: nan Test Loss: nan
Validation loss decreased (nan --> nan).  Saving model ...
Updating learning rate to 3.125e-08
>>>>>>>testing : informer_custom_ftMS_sl96_ll48_pl24_dm512_nh8_el2_dl1_df2048_atprob_fc5_ebtimeF_dtTrue_mxTrue_exp_0<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
test 353
test shape: (2, 128, 24, 319) (2, 128, 24, 1)
test shape: (256, 24, 319) (256, 24, 1)
mse:nan, mae:nan

**Comment by m13ammed** [comment]
make sure to check your dataset for NAN values or missing ones. As the script does not automatically drop NAN values. If any NAN or missing value is apparent in the dataset all values are turned into NANs during the normalization process (StandardScaler).
